app/assets/stylesheets/article-show.scss

```diff
@@ -58,10 +58,8 @@ article {
     width: 100%;
     margin: auto;
     max-width: 1024px;
-    background: transparent no-repeat center center;
-    background-size: cover;
     z-index: 2;
-    padding-top: 42%;
+    padding-top: 14%;
```

Comment:
This element will probably work if you go `padding:0` and `height:42vh`, but for media query over 880px, go `height:420px` and that should ultimately make it work.

Reply:
... Because this behavior is going to remain cached on some pages, I think we should also actually keep the `background` css here and remove it a few days after this gets merged, to make sure there are no conflicts.

So with these suggestions I'd say this can look like this....

```css
  .image {
    position: relative;
    width: 100%;
    margin: auto;
    max-width: 1024px;
    z-index: 2;
    background: transparent no-repeat center center;
    background-size: cover; 
    height: 42vw;
    @media screen and (min-width: 880px) {
      height: 370px;
    }
    @media screen and (min-width: 950px) {
      border-top-left-radius: 2px;
      border-top-right-radius: 2px;
    }
  }
```

And we'd be good to go I think.

Reply:
Okay! I've made these changes already and will add the original code back as well. 😄